docs/extensions/guides/renderer-extension.md

```diff
@@ -302,14 +305,16 @@ export default class HelpExtension extends LensRendererExtension {
 }
 ```
 
-Global page menus are objects matching the `PageMenuRegistration` interface.
-They define the appearance of the global page menu item in the cluster menu and the behaviour when the global page menu item is activated (typically by a mouse click).
-The example above uses the `target` field to set the behaviour as a link to the global page with `id` of `"help"`.
-This is done by setting `target`'s `pageId` field to `"help"`.
-The global page menu item's appearance is defined by setting the `title` field to the text that is to be displayed as a tooltip in the cluster menu.
-The `components` field is used to set an icon that appears in the cluster menu.
-Thus when the `"Help"` icon is activated the contents of `ExamplePage` will be shown.
-This example requires the definition of another React-based component, `HelpIcon`, which has been added to `page.tsx`:
+`globalPageMenus` is an array of objects that satisfy the `PageMenuRegistration` interface. This element defines how the global page menu item will appear and what it will do when you click it. The properties of the `globalPageMenus` array objects are defined as follows:
+
+* `target` links to the relevant global page using `pageId`.
+* `pageId` takes the value of the relevant global page's `id` property.
+* `title` sets the name of the global page menu item that will display as a tooltip in the left side menu.
+* `components` is used to set an icon that appears to the left of the title text in the left side menu.
+
+The above example creates a "Help" icon menu item. When users click the icon, the Lens UI will display the contents of `ExamplePage`.
+
+This example requires the definition of another React-based component, `HelpIcon`. Add the following to `page.tsx`:
```

Comment:
the following is the new page.tsx so I think it's more accurate to say "...which has been added to page.tsx:" Or "Add it to page.tsx:"